internal/model/manifest.go

```diff
@@ -95,34 +95,40 @@ func (m Manifest) Dockerignores() []Dockerignore {
 }
 
 func (m Manifest) LocalPaths() []string {
-	if sbInfo := m.StaticBuildInfo(); !sbInfo.Empty() {
-		return []string{sbInfo.BuildPath}
-	} else if fbInfo := m.FastBuildInfo(); !fbInfo.Empty() {
-		result := make([]string, len(fbInfo.Mounts))
-		for i, mount := range fbInfo.Mounts {
+	switch bd := m.DockerInfo.BuildDetails.(type) {
+	case StaticBuild:
+		return []string{bd.BuildPath}
+	case FastBuild:
+		result := make([]string, len(bd.Mounts))
+		for i, mount := range bd.Mounts {
 			result[i] = mount.LocalPath
 		}
 		return result
-	} else if dcInfo := m.DCInfo(); !dcInfo.Empty() {
-		result := make([]string, len(dcInfo.Mounts))
-		for i, mount := range dcInfo.Mounts {
-			result[i] = mount.LocalPath
+	default:
+		// TODO(matt?) DC mounts should probably be on BuildDetails, not DeployInfo
```

Comment:
...imo we should put them somewhere else entirely, but they're not relevant to the build for DC manifests cuz there is no build (that's in our control)